### PR TITLE
Spelling Fixes

### PR DESCRIPTION
**Pull request by davis-matthew**
Provides some typo fixes. Does not affect code execution

**Checklist**

Author:

- [ x] The proper base branch has been selected.  New features go on `unstable`.  Bug-fix patches can go on either `unstable` or `master`.
- [ x] Automated tests have been included in this pull request, if possible, for the new feature(s) or bug fix.  Check this box if tests are not pragmatically possible (e.g. rendering features could include screenshots or videos instead of automated tests).
- [ x] The associated GitHub issues are included (above).
- [x ] Notes have been included (above).

Reviewers:

- [ ] All automated checks are passing (green check next to latest commit).
- [ ] At least one reviewer has signed off on the pull request.
- [ ] Just after this pull request is merged, it should be applied to both the `master` branch and the `unstable` branch.  Normally, this just requires cherry-picking the corresponding merge commit from `master` to `unstable` -- or vice versa.

### CODE REVIEW

File: dist/cytoscape.cjs.js [hunk 1]
```diff
@@ -884,7 +884,7 @@ var uuid = function uuid(a, b
   b += a * 51 & 52 // if "a" is not 9 or 14 or 19 or 24
   ? //  return a random number or 4
   (a ^ 15 // if "a" is not 15
-  ? // genetate a random number from 0 to 15
```
Comment (maxkfranz) [inline dist/cytoscape.cjs.js:887]: We shouldn't be including changes to the dist files in a PR

**Review by maxkfranz — CHANGES_REQUESTED** [review]
Would you revert the changes to the dist files?  It looks like it would be good to merge in after that

File: dist/cytoscape.cjs.js (hunk 1 above)
Comment (davis-matthew) [reply dist/cytoscape.cjs.js]: Now removed from PR.

**Comment by maxkfranz** [comment]
Looks great

Thanks for the nudge

**Pull request merged**